base/install_cagateway.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+set -ex
+
+. $EPICS_IN_DOCKER/install-functions.sh
+. $EPICS_IN_DOCKER/modules_versions.sh
```

Comment:
`s/modules_versions.sh/cagateway_versions.sh/`.

That should fix CI, I hope. Maybe we should (re?)consider using `set -u`. It would have made this mistake easier to catch.

Reply:
Eh, I hadn't even built locally, so I don't think that was the only thing missing.